src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,81 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.io.*;
```

Comment:
Is there a use case for this wildcard import? Perhaps consider listing the imports explicitly as per the coding standards! 😊